lib/heroku-config-vars.js

```diff
@@ -0,0 +1,59 @@
+const herokuApi = require('./heroku-api');
+
+class HerokuConfigVars {
+
+	constructor (settings) {
```

Comment:
This would really benefit from a comment before the method that explains why we're handling the `review-app` target differently to others, including a mention that we're using a preview API and a link to any relevant Heroku docs.

JSDoc style comments are the general convention for JavaScript i.e. http://usejsdoc.org/howto-es2015-classes.html